app/src/components/layout/phaseBanner/PhaseBanner.tsx

```diff
@@ -9,7 +10,11 @@ function PhaseBanner() {
                         New Service
                     </strong>
                     <span className="govuk-phase-banner__text">
-                        Your feedback will help us to improve this service.
+                        Your{' '}
+                        <a href={routes.FEEDBACK} target="_blank" rel="noreferrer">
```

Comment:
Incorrect useage of the anchor href. These should be for external links only as it does not take advantage of the react-router and hard reloads the entire page. React router is capable of detecting what components remain, and what stays, including our context.

Instead use an onClick with e.preventDefault() + useNavigate. or use a Link to="" component. You can find examples of this around the application

Reply:
Thanks!!  just replaced the <a> tag with <Link> . Will continue to address other issues as well